content/en/monitors/service_level_objectives/_index.md

```diff
@@ -159,7 +178,7 @@ Three types of SLO audit events appear in the Event Explorer:
 
 To get a full list of all SLO audit events, enter the search query `tags:audit,slo` in the Event Explorer. To view the list of audit events for a specific SLO, enter `tags:audit,slo_id:<SLO ID>` with the ID of the desired SLO.
 
-You can also query the Event Explorer programmatically using the [Datadog Events API][13].
+You can also query the Event Explorer programmatically using the [Datadog Events API][17].
```

Comment:
This link didn't work for me (led to a 404).

Reply:
Thank you for catching this!